app/models/checkout.rb

```diff
@@ -30,16 +34,28 @@ def recalled?
     recalled_date.present?
   end
 
-  def patron_key
-    fields['patron']['key']
+  def claims_returned_date
+    Time.zone.parse(fields['claimsReturnedDate']) if fields['claimsReturnedDate']
+  end
+
+  def claims_returned?
+    claims_returned_date.present?
+  end
+
+  def renewal_date
+    Time.zone.parse(fields['renewalDate']) if fields['renewalDate']
   end
 
   def overdue?
     fields['overdue']
   end
 
-  def library
-    fields['library']['key']
+  def accrued
+    fields.dig('estimatedOverdueAmount', 'amount').to_d
+  end
```

Comment:
I'm curious about this because it is using a different method than `Fines` for determining owed amount... 🤔

Reply:
For the main test account we use the Fines are different from the "You own" on checkouts 🤔

Reply:
resolved in Slack. the you owe in checkouts is like a "pending charges" really.